startupscript/butane/idle-shutdown.sh

```diff
@@ -46,6 +39,11 @@ LAST_ACTIVE_PROXY="$(get_guest_attribute "last-active/proxy" "0")"
 if [[ "${LAST_ACTIVE}" -lt "${LAST_ACTIVE_PROXY}" ]]; then
     LAST_ACTIVE="${LAST_ACTIVE_PROXY}"
 fi
+# get the latest time between the last boot time and the last active time. The last active timestamp could still be really old when the VM is rebooted and
+# the last active timestamp is not updated yet.
+if [[ "${LAST_ACTIVE}" -lt "${UP_TIME}" ]]; then
```

Comment:
The [Google Style Guide](https://google.github.io/styleguide/shellguide.html) doesn't specify a preference:

> Use (( … )) or -lt and -gt for numerical comparison.

but readers might find this more familiar syntactically:

```
if (( LAST_ACTIVE < UP_TIME )); then
```